backend/de.metas.manufacturing/src/main/java/org/eevolution/api/PPOrderCreateRequest.java

```diff
@@ -107,7 +109,8 @@
 			@Nullable final ProjectId projectId,
 			//
 			@Nullable final Boolean completeDocument,
-			@Nullable final HUPIItemProductId packingMaterialId)
+			@Nullable final HUPIItemProductId packingMaterialId,
+			final int seqNo)
```

Comment:
it's OK for me to call the new column `OrdercandidateSeqNo` (as opposed to renaming the exiging one to `ProductPlanningSeqNo`), but then please also call the fields and params `orderCandidateSeqNo` as well, to reduce FUD when looking at both SeqNos

Reply:
LE: actually the name OrdercandidateSeqNo does not really help - according to the name the seqno of the order candidate 
...as opposed to the other column of the same table that's just named `SeqNo`.
If anything, wdyt about calling is `ProcessingSeqNo`